Production/plugins/GenJetFlavourTableProducerHLT.cc

```diff
@@ -0,0 +1,95 @@
+// system include files
+#include <memory>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+
+#include "DataFormats/JetReco/interface/GenJet.h"
+#include "DataFormats/JetMatching/interface/JetFlavourInfo.h"
+#include "DataFormats/JetMatching/interface/JetFlavourInfoMatching.h"
+#include "DataFormats/Math/interface/deltaR.h"
+
+#include "DataFormats/NanoAOD/interface/FlatTable.h"
+
+#include "CommonTools/Utils/interface/StringCutObjectSelector.h"
+#include "CommonTools/Utils/interface/StringObjectFunction.h"
+
+class GenJetFlavourTableProducerHLT : public edm::stream::EDProducer<> {
```

Comment:
doesn't need to duplicate the code. [GenJetFlavourTableProducer](https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/NanoAOD/plugins/GenJetFlavourTableProducer.cc) from CMSSW should work fine for our case